api/envoy/api/v2/cluster/circuit_breaker.proto

```diff
@@ -50,4 +50,7 @@ message CircuitBreakers {
   // :ref:`RoutingPriority<envoy_api_enum_core.RoutingPriority>`, the default values
   // are used.
   repeated Thresholds thresholds = 1;
+
+  // Whether circuit breaking should be enabled. Defaults to true.
+  google.protobuf.BoolValue enabled = 2;
```

Comment:
Instead of a field, can the absence of the object be treated as a disabled circuit breaker?

Reply:
For use cases where circuit breakers are to be disabled temporarily, this field will give the flexibility to retain the configuration in its place and do the disabling by supplying just 1 additional parameter. Thoughts?

Reply:
Yeah, but keep in mind that we are not necessarily optimizing for hand configured Envoys. Also, given that this is all YAML, its very easy to simply comment out the entire circuit breaker thing. Often times, the user will *not* configure a circuit breaker (that being her intent - no circuit breaker). So, if not configured, it should not be enabled.. It makes no sense to configure the circuit breaker with all the 100 levels of nesting in a proto, only to set a small boolean field..

Reply:
Makes sense, fixed in 86bba98.